NEWS.md

```diff
@@ -4,6 +4,7 @@
 ## Major changes
 
 - `ParameterIdentification$gridSearch()` and`ParameterIdentification$calculateOFVProfiles()` are made available and refactored for robustness, clarity, and efficiency (#151).
+- New `Optimizer` class for improved stability and extensibility (#161).
```

Comment:
Add note that this class is internal, the workflow does not change for the user.